pilot/pkg/proxy/envoy/v1/mixer.go

```diff
@@ -157,23 +201,26 @@ func mixerHTTPRouteConfig(mesh *meshconfig.MeshConfig, role model.Node, instance
 	}
 	v2 := &mccpb.HttpClientConfig{
 		MixerAttributes: &mpb.Attributes{
-			Attributes: map[string]*mpb.Attributes_AttributeValue{
-				AttrDestinationIP:  {Value: &mpb.Attributes_AttributeValue_BytesValue{net.ParseIP(role.IPAddress)}},
-				AttrDestinationUID: {Value: &mpb.Attributes_AttributeValue_StringValue{"kubernetes://" + role.ID}},
-			},
+			Attributes: map[string]*mpb.Attributes_AttributeValue{},
 		},
 		ServiceConfigs: map[string]*mccpb.ServiceConfig{},
 	}
 
+	var labels map[string]string
+	// Note: instances are all running on mode.Node named 'role'
+	// So instance labels are the workload / Node labels.
+	if len(instances) > 0 {
+		labels = instances[0].Labels
+	}
+	addStandardNodeAttributes(v2.MixerAttributes.Attributes, AttrDestinationPrefix, role, labels)
```

Comment:
The function `serviceConfig()` below already sets labels on a per-service level. Why do we need to also set them here?